### PR TITLE
2023-08-21-twpm-testing.md: intro and sections

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

File: blog/content/post/2023-08-21-twpm-testing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+This project was funded through the [NGI Assure Fund](https://nlnet.nl/assure),
+a fund established by [NLnet](https://nlnet.nl/) with financial support from the
+European Commission's [Next Generation Internet](https://ngi.eu/) programme,
+under the aegis of DG Communications Networks, Content and Technology under
+grant agreement No 957073.
+
+The project just entered a phase in which all basic components can communicate
+with each other. This should be enough to test simplest commands that don't
+require use of parts that aren't implemented yet. Nonvolatile storage, true
```
Comment (arturkow2) [inline blog/content/post/2023-08-21-twpm-testing.md:37]: Non-volatile storage already works on EOS S3

Comment (krystian-hebel) [reply blog/content/post/2023-08-21-twpm-testing.md]: This no longer applies, we switched to Orange Crab.

File: blog/content/post/2024-01-19-twpm-testing.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+This project was funded through the [NGI Assure Fund](https://nlnet.nl/assure),
+a fund established by [NLnet](https://nlnet.nl/) with financial support from the
+European Commission's [Next Generation Internet](https://ngi.eu/) programme,
+under the aegis of DG Communications Networks, Content and Technology under
+grant agreement No 957073.
+
+The project just entered a phase in which all basic components can communicate
+with each other. This should be enough to test simplest commands that don't
+require use of parts that aren't implemented yet. Nonvolatile storage, true
+randomness source and primary keys' certificates manufacturing process are yet
```
Comment (arturkow2) [inline blog/content/post/2024-01-19-twpm-testing.md:38]:
```suggestion
randomness source and primary key's certificates manufacturing process are yet
```

File: blog/content/post/2024-01-19-twpm-testing.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+Note that only top level module repository has to be cloned recursively for
+building, all of the other modules are fetched automatically either as submodule
+or by `west` command called internally during the build process. Unless you're
+up for some hacking, you probably don't need to bother with those.
+
+That said, we cheated a bit to get as much passes as possible. Some changes were
+made to various components to make it slightly faster. In order for Linux to
+detect TPM, it must return proper results to the commands executed both by OS
+and firmware. Those commands work generally, but the results is returned after a
```
Comment (arturkow2) [inline blog/content/post/2024-01-19-twpm-testing.md:63]:
```suggestion
and firmware. Those commands work generally, but the results are returned after a
```

File: blog/content/post/2024-01-19-twpm-testing.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Ideally, this should be enough for a working solution, but for deployment we
+need to connect UART and USB to (preferably) another PC in order to flash FPGA
+bitstream and TPM software.
+
+## Compiling and flashing
+
+Compiling and flashing process is described in [TwPM_toplevel repository](https://github.com/dasharo/TwPM_toplevel/#twpm_toplevel).
+Follow those instructions till the very end. You will be asked to connect the
+UART and USB to the programming PC. When doing so, it sometimes help to keep
```
Comment (arturkow2) [inline blog/content/post/2024-01-19-twpm-testing.md:116]:
```suggestion
UART and USB to the programming PC. When doing so, it sometimes helps to keep
```

**Pull request merged**